client/src/components/views/AdminPage/AdminLoginPage.tsx

```diff
@@ -1,7 +1,13 @@
 import React from "react";
+import { ButtonWithImage } from "../../ui/ButtonWithIcon/ButtonWithIcon";
 
 const AdminLoginPage = () => {
-  return <div>AdminLoginPage</div>;
+  return (
+    <div>
+      <div>AdminLoginPage</div>
```

Comment:
I didn't add aria-label because it would be the same as button content, so I thought it is unnecessary. If I'm wrong, I will add this.